test/unit/module/custom_rules/test_custom_rules.py

```diff
@@ -27,12 +27,12 @@ def setUp(self):
                 "filename": 'test/fixtures/templates/good/generic.yaml'
             }
         }
-        self.valid_rule = 'test/fixtures/templates/good/custom_rule_test.txt'
-        self.perfect_rule = 'test/fixtures/templates/good/custom_rule_perfect.txt'
-        self.invalid_op = 'test/fixtures/templates/bad/custom_rule_invalid_op.txt'
-        self.invalid_prop = 'test/fixtures/templates/bad/custom_rule_invalid_prop.txt'
-        self.invalid_propkey = 'test/fixtures/templates/bad/custom_rule_invalid_propkey.txt'
-        self.invalid_rt = 'test/fixtures/templates/bad/custom_rule_invalid_rt.txt'
+        self.valid_rule = 'test/fixtures/templates/good/custom_rules/custom_rule_test.txt'
+        self.perfect_rule = 'test/fixtures/templates/good/custom_rules/custom_rule_perfect.txt'
+        self.invalid_op = 'test/fixtures/templates/bad/custom_rules/custom_rule_invalid_op.txt'
+        self.invalid_prop = 'test/fixtures/templates/bad/custom_rules/custom_rule_invalid_prop.txt'
+        self.invalid_propkey = 'test/fixtures/templates/bad/custom_rules/custom_rule_invalid_propkey.txt'
+        self.invalid_rt = 'test/fixtures/templates/bad/custom_rules/custom_rule_invalid_rt.txt'
```

Comment:
@Birds thoughts on something like this?
```suggestion
        self.valid_rule = 'test/fixtures/custom_rules/good/custom_rule_test.txt'
        self.perfect_rule = 'test/fixtures/custom_rules/good/custom_rule_perfect.txt'
        self.invalid_op = 'test/fixtures/custom_rules/bad/custom_rule_invalid_op.txt'
        self.invalid_prop = 'test/fixtures/custom_rules/bad/custom_rule_invalid_prop.txt'
        self.invalid_propkey = 'test/fixtures/custom_rules/bad/custom_rule_invalid_propkey.txt'
        self.invalid_rt = 'test/fixtures/custom_rules/bad/custom_rule_invalid_rt.txt'
```

Reply:
That would appear to be a more enlightened approach - Will make the changes